frontend/src/utils/documentListHelper.js

```diff
@@ -347,6 +347,44 @@ export function getRowsData(rowData) {
   return data;
 }
 
+/**
+ * @method formatStringWithZeroSplitBy
+ * @summary For a given string with a specified separator it formats the number within with zeros in front if they are below 10
+ * @param {string} date
+ * @param {string} notation
+ */
+export function formatStringWithZeroSplitBy(date, notation) {
+  const dateArr = date.split(notation);
+  const frmArr = dateArr.map((itemDate, index) => {
+    return dateArr.length > 2 &&
+      itemDate &&
+      index < 2 &&
+      itemDate.length < 2 &&
+      itemDate > 0 &&
+      itemDate < 10
+      ? '0' + itemDate
+      : itemDate;
+  });
+  return frmArr.join(notation);
+}
+
+/**
+ * @method formatDateWithZeros
+ * @summary Format date with zeros if it's like dd.m.yyyy to dd.mm.yyyyy and similar for the case when / is the sepafrator
+ * @param {string} date
+ */
+export async function formatDateWithZeros(date) {
```

Comment:
pls provide unit tests for this function.

Reply:
Done